src/amo/components/CollectionList/styles.scss

```diff
@@ -0,0 +1,58 @@
+@import "~core/css/inc/mixins";
+@import "~photon-colors/photon-colors";
+@import "~ui/css/vars";
+
+.CollectionList {
+  padding: $padding-page;
+
+  @include respond-to(large) {
+    padding: $padding-page-l;
+  }
+
+  .Card-contents {
+    font-size: $font-size-m-smaller;
+  }
+}
+
+@include respond-to(large) {
+  .CollectionList-wrapper {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 0 $padding-page-l;
+    grid-template-columns: minmax(300px, 35%) 1fr;
+    margin: 10px 0 0;
+  }
+}
+
+.CollectionList-create {
+  margin-top: 18px;
+  width: 100%;
+}
+
+ul.CollectionList-listing {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 0;
+  padding: 0;
+
+  .CollectionList-collection {
+    list-style-type: none;
+  }
+
+  a:link {
+    text-decoration: none;
+
+    .CollectionList-collection-name {
+      color: $black;
+      font-size: $font-size-default;
+      font-weight: 600;
```

Comment:
We probably have a mixin for that, like `@include font-bold()` from https://github.com/mozilla/addons-frontend/blob/master/src/core/css/inc/mixins.scss